src/fsharp/TastReflect.fs

```diff
@@ -943,6 +1055,20 @@ and ReflectAssembly(g, ccu: CcuThunk, location:string) as asm =
             match x.GetType(enc) with 
             | null -> null
             | t -> t.GetNestedType(nm2,BindingFlags.Public ||| BindingFlags.NonPublic)
+        elif nm.Contains("`") then
```

Comment:
At the minute this won't work for nested generics and probably a fair few more cases, @dsyme is there something somewhere in the compiler I could reuse for this?